command/v7/share_service_command_test.go

```diff
@@ -60,4 +68,67 @@ var _ = Describe("share-service Command", func() {
 		})
 	})
 
+	Context("user is logged in", func() {
```

Comment:
Should this be "There's a space and org targeted" or something like that?

Reply:
updated the naming ✅